forgettable-frontend/src/pages/SettingsPage/SettingsPage.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import NavBar from './components/NavBar/NavBar';
+import classes from './SettingsPage.module.css';
+import {getAuth, signOut} from 'firebase/auth';
+import DarkMode from './DarkMode';
+
+function SettingsPage() {
+  const signOutHandler = ()=>{
+    const auth = getAuth();
```

Comment:
Okay a couple things after sorting out the front end auth login stuff:
- As part of the signout, we need to clear the localstorage for 'token', as well as the localstorage for the name.
- You will also want to pass in the setIsLoggedIn function from the App.js level into the Settings page component (like it is passed in to SignInPage) and call it with setIsLoggedIn(false) on a successful log out.

If you are confused about any of that, don't worry about it for this PR. We can send a follow up PR afterwards to make sure all the auth stuff is happy :)